docs/docs/1.4.1/deployments/how-multi-zone-works.md

```diff
@@ -0,0 +1,31 @@
+# About multi-zone deployments
+
+Kuma supports the use case where multiple service meshes run in different zones. This use case includes hybrid deployments where you run your services on Kubernetes and in Universal mode, on VMs or elsewhere. Your mesh environment can include multiple isolated service meshes (multi-tenancy), and workloads running in different regions, on different clouds, or in different datacenters. A zone can be a Kubernetes cluster, a VPC, or any other deployment you need to include in the same distributed mesh environment.
+
+## Components of a multi-zone deployment
+
+A multi-zone deployment includes:
+
+* The **global control plane**. Accepts connections only from zone control planes -- not from any data plane proxies. It provides configurations for the global [policies](/policies) that are applied to the data plane proxies.
+* The **zone control planes**. Accept connections from data plane proxies that are started in the same underlying zone, and connect to the global control plane to fetch policies for the data plane proxies that connect to them.
+* The **data plane proxies**. Connect to the zone control plane in the same zone.
+* The **zone ingress** Provides ingress to services in different zones. If a zone consumes services only, and does not provide them, the zone does not require this component.
```

Comment:
I think it's missing a work in "If a zone consumes services only" Did you mean "If a zone consumers only services local to its zone"?

Reply:
Zone A can consumer services only from Zone A, but Zone B can also consume services from Zone A, so I'm not sure it's the same

Reply:
And neither option correctly explains zone ingress, right? (Yes I know I wrote the original. It's wrong.) I think we can trim to

```suggestion
* The **zone ingress** Provides ingress to services in different zones.
```

Reply:
> Provides ingress to services in different zones

For me, it sounds like Ingress in zone A provides an entry point to services in zone B, C, D, which is not correct.

How about "Provides ingress to local services for different zones"

Reply:
I like the way you reworded that!